web/src/main/java/com/epam/esm/configuration/MyConfig.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.esm.configuration;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.hibernate5.HibernateTransactionManager;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+import java.util.Properties;
+
+@Configuration
+@ComponentScan("com.epam.esm")
+@EnableTransactionManagement
+
+public class MyConfig {
+
+    @Autowired
+    private Environment environment;
+
+    @Bean
+    public LocalSessionFactoryBean sessionFactory() {
```

Comment:
as properties, all beans should be placed in module, where you use them. Thus, these beans should be in "repository"

Reply:
made : moved to "repository"